templates/config.yaml

```diff
@@ -84,6 +84,7 @@ templates:
           <<: *kyma_generic_component
           path: components/console-backend-service
           since: '1.6'
+          optional: true
```

Comment:
Can you change `optional` to something else? `optional` is already a configuration value in prow jobs, so one might confuse if this sets the original `optional` value in the prow job.